javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
I might missing something, but don't see how `value.constructor && value.constructor === TypeError.constructor` is better than `value instanceof TypeError`